enhancements/local-storage/automatic-detection.md

```diff
@@ -202,21 +356,22 @@ type LocalVolumeGroupStatus struct {
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
 }
 ```
-### Workflow of LocalPV creation via LocalVolumeGroup CR
-- Once the `LocalVolumeGroup` is created, the controller will:
+### Workflow of `LocalPV` creation via `LocalVolumeSet CR`
+- Once the `LocalVolumeSet` is created, the controller will:
   - configure the local-static-provisioner to make a new StorageClass based on certain directories on the selected nodes.
-  - assign diskmaker daemons to the selected nodes.
+  - assign diskmaker daemonset to the selected nodes.
+  - there will be one daemonset for each `LocalVolumeSet` CR.
 - The diskmaker daemon will find devices that match the disovery policy and symlink them into the directory that the local-static-provisioner is watching.
 
 #### Note: There is a chance of race condition and duplicate creation of PVs 
-- If two `LocalVolumeGroup` CR targets same nodes with overlapping inclusion filter.
-- If `LocalVolumeGroup` CR and `LocalVolume` CR targets the same node with overlapping devices.
+- If two `LocalVolumeSet` CR targets same nodes with overlapping inclusion filter.
+- If `LocalVolumeSet` CR and `LocalVolume` CR targets the same node with overlapping devices.
```

Comment:
why have we changed the name of the CR again? I think LocalVolumeGroup was more straightforward I think

Reply:
Umm I think @jsafrane had this suggestion and we all agreed on this.

Reply:
LocalVolumeGroup resembles LVM and their volume groups, which can be confusing.